examples/model_assisted_labeling/tiled_imagery_mal.ipynb

```diff
@@ -35,34 +35,34 @@
   },
```

Comment:
<pre class="ql-syntax" spellcheck="false">Line #9.    label.assign_feature_schema_ids(ontology)
</pre><p>Maybe just do </p><p> </p><p><code>label = Label(</code></p><p><code>&nbsp;&nbsp;data=tiled_image_data,</code></p><p><code>&nbsp;&nbsp;annotations = [</code></p><p><code>&nbsp;&nbsp;&nbsp;&nbsp;ObjectAnnotation(value=Polygon(points=geo_polygons),feature_schema_id=schema_lookup["water"])</code></p><p><code>&nbsp;&nbsp;]</code></p><p><code>)</code></p><p> </p><p>Then you can get rid of the label.assign_feature_schema_ids. I think that line confuses people. </p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/Labelbox/labelbox-python/pull/374/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='21'/>